packages/opentelemetry-tracer-web/examples/js/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "example-web-tracer",
```

Comment:
What do you think about keeping this example in top `examples/` folder? Similar to other examples?

Reply:
I think it makes sense